README.md

```diff
@@ -0,0 +1,14 @@
+# AtoM documentation
+
+[AtoM](https://www.accesstomemory.org/) is a web-based, open source application for standards-based archival description and access in a multilingual, multi-repository environment.
```

Comment:
Should we list the full project name first? i.e.:

```
Access to Memory ([AtoM](https://www.accesstomemory.org/)) is a ...

```

Reply:
I wondered about that too - yes I think we should